packages/zosfiles/src/api/methods/get/Get.ts

```diff
@@ -63,11 +75,11 @@ export class Get {
      * @param {string}           USSFileName  - contains the data set name
      * @param {IViewOptions} [options={}] - contains the options to be sent
      *
-     * @returns {Promise<Buffer>} Promise that resolves to the content of the uss file
+     * @returns {Promise<any>} Promise that resolves to the content of the uss file
      *
      * @throws {ImperativeError}
      */
-    public static async USSFile(session: AbstractSession, USSFileName: string, options: IGetOptions = {}): Promise<Buffer> {
+    public static async USSFile(session: AbstractSession, USSFileName: string, options: IGetOptions = {}): Promise<any> {
```

Comment:
Let's try to avoid `any` in the future.

Reply:
fixed with https://github.com/zowe/zowe-cli/pull/588/commits/4d836a3c22d6e6bfd7d451d7b162301979a0a9cb